src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,280 @@
+<!-- If using debug menu shortcuts, prompts player to enter their name before beginning initial Roach Market sequence -->
+::RM-first-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 1)[Right, my name is $playerName. [[Moving on|RM-first-entrance]].<!-- what this code does is check the ending of the custom $playerName string to hopefully pluralize it properly -->{
+  (if: $playerName's last is "s")[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
```

Comment:
Good to know about `last` and `2ndlast`! Super handy.

Reply:
Yeah those I found in the Harlowe docs. They say it "continues in that fashion" so not sure if you can just keep going `3rdlast`, `4thlast` foreva